<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="4" ignored="4" total="15" passed="5" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-03T17:32:04 MDT" name="Suite" finished-at="2024-09-03T17:35:04 MDT" duration-ms="180304">
    <groups>
    </groups>
    <test started-at="2024-09-03T17:32:04 MDT" name="Test" finished-at="2024-09-03T17:35:04 MDT" duration-ms="180304">
      <class name="RC.Tests.SubmitOrder">
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:04 MDT" name="initializeDriver" finished-at="2024-09-03T17:32:07 MDT" duration-ms="2776" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="checkIfTheProductsAreAvailable()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:07 MDT" name="checkIfTheProductsAreAvailable" finished-at="2024-09-03T17:32:19 MDT" duration-ms="12315" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIfTheProductsAreAvailable -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:19 MDT" name="tearDown" finished-at="2024-09-03T17:32:20 MDT" duration-ms="690" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:20 MDT" name="initializeDriver" finished-at="2024-09-03T17:32:22 MDT" duration-ms="1749" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:22 MDT" name="submitOrder" data-provider="data" finished-at="2024-09-03T17:32:43 MDT" duration-ms="21646" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3, IPHONE 13 PRO], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:43 MDT" name="tearDown" finished-at="2024-09-03T17:32:44 MDT" duration-ms="553" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:44 MDT" name="initializeDriver" finished-at="2024-09-03T17:32:45 MDT" duration-ms="1319" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:32:45 MDT" name="submitOrder" data-provider="data" finished-at="2024-09-03T17:33:04 MDT" duration-ms="19139" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.SubmitOrder@45385f75]" started-at="2024-09-03T17:33:04 MDT" name="tearDown" finished-at="2024-09-03T17:33:05 MDT" duration-ms="804" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- RC.Tests.SubmitOrder -->
      <class name="RC.Tests.CartTests">
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:06 MDT" name="initializeDriver" finished-at="2024-09-03T17:33:07 MDT" duration-ms="1854" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method retried="true" signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:07 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:33:20 MDT" duration-ms="12530" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3, IPHONE 13 PRO], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [263000] but found [-263000]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [263000] but found [-263000]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:20 MDT" name="tearDown" finished-at="2024-09-03T17:33:20 MDT" duration-ms="362" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:20 MDT" name="initializeDriver" finished-at="2024-09-03T17:33:22 MDT" duration-ms="1269" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method retried="true" signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:22 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:33:35 MDT" duration-ms="13194" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3, IPHONE 13 PRO], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [263000] but found [-263000]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [263000] but found [-263000]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:35 MDT" name="tearDown" finished-at="2024-09-03T17:33:35 MDT" duration-ms="337" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:35 MDT" name="initializeDriver" finished-at="2024-09-03T17:33:37 MDT" duration-ms="1441" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:37 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:33:50 MDT" duration-ms="12941" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3, IPHONE 13 PRO], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [263000] but found [-263000]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [263000] but found [-263000]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:50 MDT" name="tearDown" finished-at="2024-09-03T17:33:50 MDT" duration-ms="352" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:50 MDT" name="initializeDriver" finished-at="2024-09-03T17:33:52 MDT" duration-ms="1429" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method retried="true" signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:33:52 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:34:03 MDT" duration-ms="11736" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [31500] but found [-31500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [31500] but found [-31500]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:03 MDT" name="tearDown" finished-at="2024-09-03T17:34:04 MDT" duration-ms="327" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:04 MDT" name="initializeDriver" finished-at="2024-09-03T17:34:05 MDT" duration-ms="1619" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method retried="true" signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:05 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:34:18 MDT" duration-ms="13017" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [31500] but found [-31500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [31500] but found [-31500]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:18 MDT" name="tearDown" finished-at="2024-09-03T17:34:19 MDT" duration-ms="392" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:19 MDT" name="initializeDriver" finished-at="2024-09-03T17:34:20 MDT" duration-ms="1418" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="checkTotal(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:20 MDT" name="checkTotal" data-provider="data" finished-at="2024-09-03T17:34:32 MDT" duration-ms="12166" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [31500] but found [-31500]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [31500] but found [-31500]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at org.testng.Assert.assertEquals(Assert.java:1429)
at RC.Tests.CartTests.checkTotal(CartTests.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkTotal -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:33 MDT" name="tearDown" finished-at="2024-09-03T17:34:33 MDT" duration-ms="348" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:33 MDT" name="initializeDriver" finished-at="2024-09-03T17:34:34 MDT" duration-ms="1332" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="checkCartProductsSameAsFoundItems(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:34 MDT" name="checkCartProductsSameAsFoundItems" data-provider="data" finished-at="2024-09-03T17:34:49 MDT" duration-ms="14892" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3, IPHONE 13 PRO], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCartProductsSameAsFoundItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:49 MDT" name="tearDown" finished-at="2024-09-03T17:34:51 MDT" duration-ms="1974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="initializeDriver()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:51 MDT" name="initializeDriver" finished-at="2024-09-03T17:34:53 MDT" duration-ms="1469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initializeDriver -->
        <test-method signature="checkCartProductsSameAsFoundItems(java.util.HashMap)[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:34:53 MDT" name="checkCartProductsSameAsFoundItems" data-provider="data" finished-at="2024-09-03T17:35:04 MDT" duration-ms="11474" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{searchWords=[ZARA COAT 3], password=Rohitha@26, userId=rohitha03@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCartProductsSameAsFoundItems -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:RC.Tests.CartTests@769a1df5]" started-at="2024-09-03T17:35:04 MDT" name="tearDown" finished-at="2024-09-03T17:35:04 MDT" duration-ms="380" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- RC.Tests.CartTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
